### PR TITLE
Associate grammar file with CodeLens

### PR DESCRIPTION
**Pull request by angelozerr**
This PR provides the capability to associate an XML file with a XSD, DTD, RelaxNG grammar file easily by clicking on the CodeLens. See the following demo:

WWDAssociateGrammarDemo

### CODE REVIEW

**Comment by angelozerr** [comment]
CI Build failed, but I don't understand, please help me.

**Comment by mickaelistria** [comment]
First, please ignore the windowsPR build, it suffers from a bug that makes it not reliable. Then, please don't change bundle version of bundles that are not modified by your patch.

File: org.eclipse.wildwebdeveloper.xml/pom.xml [hunk 1]
```diff
@@ -4,10 +4,10 @@
 	<parent>
 		<groupId>org.eclipse.wildwebdeveloper</groupId>
 		<artifactId>parent</artifactId>
-		<version>1.0.0-SNAPSHOT</version>
+		<version>1.0.1-SNAPSHOT</version>
```
Comment (vrubezhny) [inline org.eclipse.wildwebdeveloper.xml/pom.xml:7]: Please, roll it back to be `<version>1.0.0-SNAPSHOT</version>` - we haven't change parent version yet

**Review by vrubezhny — COMMENTED** [review]
Could you, please:
1) rebase against latest master
2)  bump org.eclipse.wildwebdeveloper.feature  to be 1.0.1-SNAPSHOT
and forcibly push these changes

Thanks

**Pull request merged**